csrc/aio/py_lib/deepspeed_py_copy.cpp

```diff
@@ -10,7 +10,7 @@ Functionality for swapping optimizer tensors to/from (NVMe) storage devices.
 #include "deepspeed_py_copy.h"
 #include <omp.h>
 
-#define ROUND_DOWN(size, step) ((size) & ~((step)-1))
+#define ROUND_DOWN(size, step) ((size) & ~((step) - 1))
```

Comment:
Is this related?

Reply:
I think this is due to formatting, we probably should update the clang-format version we use too.